src/modeler/AutoRest.Swagger/Validation/Extensions/LongRunningExtensionRule.cs

```diff
@@ -100,7 +87,7 @@ private static bool IsPostResponseCodesValid(Dictionary<string, OperationRespons
         /// </summary>
         /// <param name="responses">Dictionary of responses for the operation.</param>
         /// <returns><c>true</c> if at least one valid terminal status code availabe, otherwise <c>false</c>.</returns>
-        private static bool IsDeleteResponseCodesValid(Dictionary<string, OperationResponse> responses)
+        protected static bool IsDeleteResponseCodesValid(Dictionary<string, OperationResponse> responses)
```

Comment:
is the condition and code block in line 95 the same as in line 100? Should the method name be "AreDeleteResponseCodesValid" (it seems that there's an inconsistency in plural and singular)?